mne/io/egi/egimff.py

```diff
@@ -211,24 +210,27 @@ def _read_header(input_fname):
     mff_hdr = _read_mff_header(input_fname)
     with open(input_fname + '/signal1.bin', 'rb') as fid:
         version = np.fromfile(fid, np.int32, 1)[0]
-    # This should be equivalent to the following, but no need for external dep:
-    # import dateutil.parser
-    # time_n = dateutil.parser.parse(mff_hdr['date'])
-    dt = mff_hdr['date'][:26]
-    assert mff_hdr['date'][-6] in ('+', '-')
-    sn = -1 if mff_hdr['date'][-6] == '-' else 1  # +
-    tz = [sn * int(t) for t in (mff_hdr['date'][-5:-3], mff_hdr['date'][-2:])]
-    time_n = datetime.datetime.strptime(dt, '%Y-%m-%dT%H:%M:%S.%f')
-    time_n = time_n.replace(tzinfo=_FixedOffset(60 * tz[0] + tz[1]))
```

Comment:
@scott-huberty I pushed to your fork here but I see that now the private class _FixedOffset
is not used anymore. It seems it was added by @larsoner. See

https://github.com/mne-tools/mne-python/blame/main/mne/io/egi/egimff.py#L179 in git blame

@larsoner thoughts?

Reply:
I see, I wasn't sure exactly what `_fixed offset` does but Eric mentioned that originally `meas_date` was a tuple and not a datetime object, so I was wondering if this code was needed when the meas_date was a tuple.

After having a think about it, other parts of the prior code that I removed could also be integrated to the proposed code. 
after `if len(mff_hdr['date']) >32` You could add the `assert mff_hdr['date'][-6] in ('+', '-')` from the prior code, just to be sure that you will be grabbing the time zone in the event that you have to execute this if statement. But as far as I can see from the files I've checked, any cases where the length of the timestamp was longer than the standard 32 characters, it was due to the milliseconds portion containing trailing zeros.  The month/day/hour/minute/second/offset portions of the string always appear to be zero padded in the event that they are a single-digit numbers, so they should be standard across all files.

Reply:
I think it's okay to get rid of `_FixedOffset`